resources/assets/js/app.vue

```diff
@@ -280,54 +280,52 @@
      *
      * @source https://github.com/vuejs/vue/blob/dev/src/filters/array-filters.js
      */
-    Vue.filter('caseInsensitiveOrderBy', (arr, sortKey, reverse, subSortKey) => {
+    Vue.filter('caseInsensitiveOrderBy', (arr, sortKey, reverse) => {
         if (!sortKey) {
             return arr;
         }
 
         let order = (reverse && reverse < 0) ? -1 : 1;
 
+        function compareRecordsByKey(a, b, key) {
+            let aKey = Vue.util.isObject(a) ? Vue.parsers.path.getPath(a, key) : 0;
+            let bKey = Vue.util.isObject(b) ? Vue.parsers.path.getPath(b, key) : 0;
+
+            if (_.isNumber(aKey) && _.isNumber(bKey)) {
+                return aKey === bKey ? 0 : aKey > bKey;
+            }
+
+            aKey = aKey === undefined ? aKey : aKey.toLowerCase();
```

Comment:
This line triggered a "call to toLowerCase() on null` error in your previous PR. I didn't check exactly where the error came from, but changed it into a loose `aKey ? `, as a true fan of voodoo programming, worked. Care to explore a bit?